src/components/UsercentricsProvider.tsx

```diff
@@ -28,27 +25,19 @@ interface UsercentricsProviderProps {
     windowEventName: string
 }
 
-/**
- * The provider component for listening to events from the Usercentrics Browser API.
- * Render this once and wrap your application in it.
- *
- * @example <caption>Basic usage</caption>
- * () => <UsercentricsProvider windowEventName="ucEvent"><App /></UsercentricsProvider>
- *
- * @example <caption>Custom timeout in milliseconds</caption>
- * () => <UsercentricsProvider timeout={100} windowEventName="ucEvent"><App /></UsercentricsProvider>
- */
 export const UsercentricsProvider: FC<UsercentricsProviderProps> = ({
     children,
     strictMode = false,
-    timeout = 5000,
+    timeout = 30000,
```

Comment:
In my opinion 30 seconds is too long in production. Since this can be adjusted via the `prop` for development purposes, is this a necessary change?

```tsx
<UsercentricsProvider timeout={isProduction ? 5000 : 30000}>
```

Reply:
i think the same happens occasionally in prod, the loading can take a while...

Reply:
maybe we reduce this to 15s?

Reply:
Can we just remove this change from the PR? It's technically unrelated to the main feature, and the value can already be overridden via the prop.

Reply:
whatever, other projects will have to take this into account as well.